src/Migrator.php

```diff
@@ -92,7 +92,7 @@ public function rollbackAll(): self
         return $this;
     }
 
-    public function showStatus(): self
+    public function showStatus($onlyPending = false): self
```

Comment:
Shall we make the output similar to Laravel's command? We can rewrite the method to something like this:

```php
public function showStatus(bool $onlyPending = false): self 
{
        $files = $this->migrationStorage->all();
        $migratedFiles = $this->migrationRepository->all();
        $lastBatch = $this->migrationRepository->lastBatch();
        
        $rows = $files->map(
            static fn (MigrationFile $file) => [
                $file->name(),
                $migratedFiles->contains($file->name()) 
                     ? '<fg=green;options=bold>Ran</>' . ($lastBatch->contains($file->name()) ? ' [last batch]'  : '') 
                     : '<fg=yellow;options=bold>Pending</>'
            ]
        )->when($onlyPending, static fn (Collection $rows) => $rows->filter(
            static fn (array $row) => strpos($row[1], 'Pending') !== false
        }));
        
        if ($rows->isNotEmpty()) {
            $this->newLine();
   
            $this->components->twoColumnDetail('<fg=gray>Migration name</>', '<fg=gray>Status</>');
        
            $rows->each(
                 fn ($migration) => $this->components->twoColumnDetail($rows[0], $rows[1])
             );
        
            $this->newLine();
        } elseif ($onlyPending) {
            $this->components->info('No pending migrations');
        } else {
            $this->components->info('No migrations found');
        }
}
```

I did not test this piece of code, but it looks more straightforward and similar to Laravel's implementation.

Reply:
updated ;-)

Reply:
@babenkoivan MR updated ;-)